File: Software Development Module/Week5/Cinema.java
/*
Cinema.java
Billy Cussen
13/07/2020
*/

public class Cinema {
    
    //Data Members
    private int age;
    private String result;

    //Constructor
    public Cinema(int age){
        this.age = age;
        this.result = new String();
    }

    //Compute
    public void computePrice(){

        if(this.age <= 5 || this.age >= 65){
            this.result = "You get in for Free!";
        } else if(this.age > 5 || this.age <= 14){
            this.result = "Your ticket costs 10";
        } else{
            this.result = "Your ticket costs 15";
        }

    }

    //Getter
    public String getPrice(){
        return this.result;
    }

}

File: Python/Software Development Module - Python/Week5/Cinema.py
"""
Cinema.py
Billy Cussen
27/09/2020
"""

class myCinema:
    def __init__(self,age):
        self.age = age

    def calculateTicketPrice(self):
        if self.age <= 5 or self.age >= 65:
            return "You get in for Free!"
        elif self.age > 5 and self.age <= 14:
            return "Your ticket costs: 5.50"
        else:
            return "Your ticket costs: 10.50"
